.orchestra/support/vscode-web/product.json

```diff
@@ -15,7 +15,14 @@
         "window.menuBarVisibility": "classic",
         "editor.foldingMaximumRegions": 65000,
         "editor.colorTheme": "revng",
-        "workbench.colorTheme": "revng"
+        "workbench.colorTheme": "revng",
+        "revng.general.profiles": [
+            {
+                "id": "9b473245-159e-4984-a052-f2d9131314ea",
+                "name": "Local",
+                "directURLs": ["http://127.0.0.1:8000/"]
```

Comment:
The only other place to put this would be in `revng-ui` as the default for the options, but I don't know if it's a sensible thing to do (since this is the config for the `revng web-ui` command)

Reply:
Uhm OK.